pkgs/development/ocaml-modules/janestreet/0.14.nix

```
@@ -773,9 +773,9 @@ with self;
 
   spawn = janePackage {
     pname = "spawn";
-    version = "0.13.0";
+    version = "0.15";
```

Nit:
```suggestion
    version = "0.15.0";
```
since this is a bot PR, can be a follow-up